### DISCUSSION TITLE
Plugin global configuration

### DISCUSSION THREAD

**Discussion by splisp** [iPlug2/iPlug2#1090] [General]
Hi, is there a best practice for storing a plugin's global configuration? I am referring to things such as preferred import directories, UI settings, and the likes; settings that are shared amongst all the plugin's instances, and the standalone app as well. Is there any method provided by the framework for this?

TIA
Fabrizio

**Comment by olilarkin** [comment]
you can use the win32/SWELL Get/SetPrivateProfileString()

example:
https://gist.github.com/olilarkin/4c9b5c9697b5c5720017e7b5c8f030c9

**Reply by splisp** [reply]
Thanks @olilarkin, just what I had in mind. Also, that feature on NAM (remember last Model and IR folders) would be really useful!

**Reply by olilarkin** [reply]
agreed!
